t.py

```diff
@@ -79,7 +91,7 @@ class RegisterOperand(Operand):
 
     def __init__(self, name):
         self.name = name.lower()
-        self.size()
+        self.size()  # TODO: why call this here?
```

Comment:
I think this call is unnecessary, should I remove it?

Reply:
The call is used for detecting invalid registers, so when passing a wrong name to the constructor the size() call will throw an exception